### DISCUSSION TITLE
How to check my student package term of validity?

### DISCUSSION THREAD

**Discussion by zlaazlaa** [community/community#133345] [GitHub Education] [answered]
### Hi everyone,

I saw a date in my billing
![image](https://github.com/user-attachments/assets/82b446b6-8e5c-4b85-b9e3-13dad5ce252b)
Will my student package renew automatically in August 09, 2024 (which is the next day of my last day)

**Comment by davevad93** [comment, marked as answer]
Hi @zlaazlaa ,

Student benefits last for 2 years, after which you will need to revalidate your student status by going to the [application page](https://education.github.com/discount_requests/application). However your benefits will expire on Oct  7, 2024 not on Aug 9, 2024.

**Reply by zlaazlaa** [reply]
> Hi @zlaazlaa ,
> 
> Student benefits last for 2 years, after which you will need to revalidate your student status by going to the [application page](https://education.github.com/discount_requests/application). However your benefits will expire on Oct 7, 2024 not on Aug 9, 2024.

Thanks for your help, I will revalidate on Oct 6, 2024.👍❤️

**Reply by davevad93** [reply]
You're welcome @zlaazlaa 👍
